src/Symfony/Component/Ldap/Security/AssignDefaultRoles.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Security;
+
+use Symfony\Component\Ldap\Entry;
+
+final readonly class AssignDefaultRoles implements RoleFetcherInterface
+{
+    /**
+     * @param string[] $roles
+     */
+    public function __construct(private array $roles)
```

Comment:
```suggestion
    public function __construct(
    private array $roles)
```

newline please

Reply:
Hi,
I modified this file that now have arguments on a dedicated line.